### ISSUE TITLE
Which data we need to use ?

### ISSUE THREAD

**Issue by thegodone** [rxn4chemistry/rxn-onmt-models#7]
I wonder to understand the datasets that are needs and according sources if available:
- USPTO ?
- USPTO-stereo ?
- USPTO-MIT ?
- ...

**Comment by avaucher** [comment]
This GitHub repository is independent of any specific dataset; any of the datasets you mentioned can work.

Or was your question specific to a publication that relied on this repository?

**Comment by thegodone** [comment]
Indeed, it was more related to data process and a publication reference: 
What is the best way to do it ? 
For example, let say with Pistachio, I need to use the atommap version or not ? etc... 
That would be nice to have a demo of the data process on few reactions.

**Comment by avaucher** [comment]
Hard to give a precise answer because some of the papers relied on previous and more "manual" versions of the code.

As an example, IIRC the atom-maps are removed by default.

My suggestion would be to start with https://github.com/rxn4chemistry/rxn-onmt-models?tab=readme-ov-file#the-easy-way, run the commands as provided, and see if what comes out is what you expected. 

Calling the commands with `--help` also shows some useful options. 

Let me know if this helps! Happy to help more, but I currently can't do much more than providing best-effort support.

**Comment by thegodone** [comment]
Thanks Alain,
Maybe simpler option is to ask for the data for "Completion of partial chemical equations" paper as a start as suggeted in the Data availability statement section ? 
Best Guillaume

**Comment by avaucher** [comment]
Hi Guillaume,
Oh, in this case I'm of little help (not at IBM anymore). I'd suggest reaching out to Federico (email address in the paper) to ask for details.
Having said that, any dataset of sufficient size and quality should work.
